libraries/tracking/subscriptions/checkout.js

```diff
@@ -11,8 +11,6 @@ import { checkoutDidEnter$ } from '../streams/checkout';
  */
 export default function checkout(subscribe) {
   subscribe(checkoutDidEnter$, ({ getState }) => {
-    setPWAVisibleState(false);
```

Comment:
This change affects the `pageview` events, which are dispatched for users who where not logged in when they entered the checkout. For them the system will now track an additional pageview for the `cart` right after `initiatedCheckout`.
That happens because `APP_EVENT_VIEW_DID_DISAPPEAR` is triggered after the `history.pop()` which is called to remove the login page from the PWA history stack.